modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -1314,6 +1314,7 @@ aggregation-field-input-help=Follow the steps in the documentation to use object
 aggregation-settings=Aggregation Settings
 aggregation-type=Aggregation Type
 aggregation-type-help=Select the aggregation type
+ai-assistent=AI Assistant
```

Comment:
@DavysonMelo rename to ai-assist**a**nt and run `gradlew buildLang`, put the generated files in a separeted commit.

Thanks!